docs/docs/external/getting-started/enabling-weaver-network/fabric.md

```diff
@@ -404,17 +715,114 @@ To prepare your network for interoperation with a foreign network, you need to r
   | Notes |
   |:------|
   | For any cross-network data request, make sure an access control policy is recorded in the _source network_ (`trade-logistics-network` in the above example) and a corresponding verification policy is recorded in the _destination network_ (`trade-finance-network` in the above example) before any relay request is triggered. |
-- **Foreign network security domain (membership) configuration**:
-  Run the following procedure (pseudocode) to record security domain configuration for every foreign network you wish your Fabric network to interoperate with (you will need to collect the identity service URLs for all the foreign networks first):
-  ```
-  for each foreign network:
-      send an HTTP GET request to the network's identity service (using 'curl' or 'wget' from a shell script or equivalent programming language APIs)
-      invoke `CreateMembership` or `UpdateMembership` on the Fabric Interoperation Chaincode with the above HTTP response as argument
-  ```
-  As in the above two cases, use `CreateMembership` to record a confiuration for the first time for a given `securityDomain` and `UpdateMembership` to overwrite a configuration.
-
-  | Notes |
-  |:------|
-  | Security domain configurations (organization lists and their certificate chains) for any Fabric network channel are subject to change, so you should run the above procedure periodically in a loop. |
+- **Local network security domain (membership) configuration**:
+  Call the function or HTTP endpoint to record local membership that was created in development step.
 
 Your Fabric network is now up and running with the necessary Weaver components, and your network's channel's ledger is bootstrapped with the initial configuration necessary for cross-network interactions!
+
```

Comment:
Below lines are all commented out, and I saw them earlier in the instructions. can you just delete them?

Reply:
Yeah wanted to ask, currently I've only listed all chaincode and API functions with `contractId`.
Others I skipped but had it in the comment, in case you think we should add those also. Its just it'd make the page too long, and probably they won't need to know all of them (only one of them either all contractIds or without contractIds)

Reply:
OK. Then further above, just add a line saying that there is another API that an app developer can use, which doesn't involve contract IDs. For details, point them to the READMEs.

Reply:
Okay moved those commented text in README, and added a link here.